recipes/apbs/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "apbs" %}
+{% set version = "1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+    git_rev: apbs-{{ version }}
+    git_url: https://github.com/Electrostatics/apbs-pdb2pqr.git
+
+build:
+    number: 0
```

It looks like you don't include a `build.bat` file so Windows should be skipped
```suggestion
    number: 0
    skip: true  # [win]
```